src/utils.js

```diff
@@ -1460,6 +1460,81 @@ function getSpanException(error) {
     return error;
 }
 
+/**
+ * Builds and bundles orders which their details are queried from a orderbook subgraph
+ *
+ * @param {any[]} ordersDetails - Orders details queried from subgraph
+ * @param {boolean} _shuffle - To shuffle the bundled order array at the end
+ * @param {boolean} _bundle = If orders should be bundled based on token pair
+ * @returns Array of bundled take orders
+ */
+const bundleOrders = (
+    ordersDetails,
+    _shuffle = true,
+    _bundle = true,
+) => {
+    const bundledOrders = [];
+    for (let i = 0; i < ordersDetails.length; i++) {
+        const orderDetails = ordersDetails[i];
+        const orderStruct = JSON.parse(ordersDetails[i].orderJSONString);
```

Comment:
why can't we use the orderStruct directly? why do we need this other _output thing?

Reply:
what do you mean by _output? becasue _output is the validoutput of the orderStruct, 
if i understand correctly, you meant the orderDetails? in which case it is just used for getting the token symbol, thats all to it